frontend/app/routes/$lang/_public/status/$id/myself.tsx

```diff
@@ -121,13 +120,19 @@ export async function action({ context: { session }, params, request }: ActionFu
   const statusId = await applicationStatusService.getStatusIdWithSin({ sin, applicationCode: code });
   const clientFriendlyStatus = statusId ? lookupService.getClientFriendlyStatusById(statusId) : null;
 
-  return json({
-    status: {
-      ...(clientFriendlyStatus ? localizeClientFriendlyStatus(clientFriendlyStatus, locale) : {}),
-      alertType: getContextualAlertType(statusId),
+  saveStatusState({
+    params,
+    session,
+    state: {
+      statusCheckResult: {
+        ...(clientFriendlyStatus ? localizeClientFriendlyStatus(clientFriendlyStatus, locale) : {}),
+        alertType: getContextualAlertType(statusId),
+        statusId: statusId,
+      },
```

Comment:
Should we be saving the status state in `/status/$id/child` as well?

Reply:
Extended to child status check